packages/fluent-theme/src/fluent/styles/SearchBox.styles.ts

```diff
@@ -1,14 +1,44 @@
 import { ISearchBoxStyleProps, ISearchBoxStyles } from 'office-ui-fabric-react/lib/SearchBox';
 
 export const SearchBoxStyles = (props: ISearchBoxStyleProps): Partial<ISearchBoxStyles> => {
-  const { theme } = props;
+  const { theme, underlined } = props;
   const { effects, palette } = theme;
 
   return {
     root: [
       {
         borderRadius: effects.roundedCorner2,
-        borderColor: palette.neutralSecondaryAlt
+        borderColor: palette.neutralTertiary
+      },
+      underlined && [
+        'is-underlined',
+        {
+          borderRadius: '0px'
+        }
+      ]
+    ],
+    iconContainer: [
+      'ms-SearchBox-iconContainer',
+      {
+        color: palette.themeDarkAlt
+      }
+    ],
+    clearButton: [
+      'ms-SearchBox-clearButton',
+      {
+        selectors: {
```

Comment:
this entire selector feels wrong... If I understand the idea, you wanted to style the clearButton for its hover state, so why not having the hover selector nest the rest things. Let me know and I can help.